packages/core/src/components/SingleSelect/types.ts

```diff
@@ -59,6 +60,8 @@ export interface SelectProps extends InputProps, WithThemeProp {
     validator?: (val: any) => string;
     /** Show Helper text and error text in a tooltip (popover) */
     showTooltipForHelperAndErrorText?: boolean;
+    /** Toggle icon */
+    suffix?:  React.FC<any>;
```

Comment:
@gmukul01 can we narrow down the type of `suffix` to `SvgIconProps` or `HTMLOrSVGElement` instead of keeping it open with `any` ?

Reply:
Yes, that's a good callout @wsameer . I had used `any`, similar to the type of `prefix`, but I also prefer to specify the `suffix` props type as `SvgIconProps`, which are all optional. I've made that change, but of course will wait for @gmukul01 's confirmation.

Reply:
Looks good now.